### DISCUSSION TITLE
Create a TouchableOpacity core-component like in React-Native

### DISCUSSION THREAD

**Discussion by relez** [NativeScript/rfcs#37] [1. Ideas & Discussions]
I have been using Label component to create custom buttons in NS but it does not have the ability to create a touchable effect like in React-Native via css. A component like this would be great to create custom and nice Buttons in iOS/Android.

**Comment by edusperoni** [comment]
I believe my other rfc encompasses this one. Please take a look: https://github.com/NativeScript/rfcs/discussions/13
